### ISSUE TITLE
Add a reminder in the readme that sending unsolicited DM advertising is against both Discord's ToS and the Canadian Anti-Spam Legislation

### ISSUE THREAD

**Issue by fffelix-jan** [alexlyee/massdm#28]
I am fed up with people using such bots for spamming purposes, and I would like to see such a reminder to discourage the program's use for malicious purposes.

**Comment by alexlyee** [comment]
Good idea!! The way this bot is designed, that shouldn't be possible without significant modification of the source code, but in any case, I'll definitely clarify that in my next update.

**Comment by alexlyee** [comment]
I'm also sick of people DMing me asking me to 'fix it' or 'help them' when they either have no clue how to use a Discord bot or only want a userbot to massDM people in a server they don't even own

**Comment by fffelix-jan** [comment]
Thanks!

**Comment by fffelix-jan** [comment]
You can simply write a disclaimer in README.md. Warnings within the program itself would be appreciated, but it would just be a plus.

**Comment by alexlyee** [comment]
I'd like to rewrite the readme and the welcome message in the next revision anyways. Feel free to make a PR if you want it done sooner --I'd love to give you credit for your contribution if it is significant enough!
